streaming/receiver.c

```diff
@@ -719,10 +722,12 @@ static int rrdpush_receive(struct receiver_state *rpt)
     rrdcontext_host_child_disconnected(rpt->host);
 
 #ifdef ENABLE_ACLK
-    // in case we have cloud connection we inform cloud
-    // new child connected
-    if (netdata_cloud_setting)
-        aclk_host_state_update(rpt->host, 0);
+    // Child disconnection
+    if (netdata_cloud_setting) {
+        struct aclk_database_worker_config *wc = (struct aclk_database_worker_config *)rpt->host->dbsync_worker;
+        if (!wc || wc->update_node_after == -1)
+            wc->update_node_after = now_realtime_sec() + ACLK_NODE_UPDATE_DELAY * 3;
```

Comment:
If `wc` is `NULL` (the first part of the condition) we can't dereference it.

Reply:
What was I thinking.. :smiley:  will fix